preupg/creator/ui_helper.py

```diff
@@ -195,6 +190,24 @@ def get_content_info(self):
             desc = get_user_input(settings.content_desc_text, any_input=True)
             self.content_dict['content_description'] = desc
 
+    @staticmethod
+    def _write_to_file_content(file_path, config):
+        """
+        Create config file
+
+        @param {str} file_path - path of new config file
+        @param {RawConfigParser} config - configuration object with config data
+        @throws {IOError}
+        """
+        try:
+            f = open(file_path, 'wb')
+            config.write(f)
+            f.close()
+        except IOError:
+            print('We have a problem with writing {0} file to disc'.format(
```

Comment:
disc -> disk
OR
remove the "to disc" .. you don't know, what kind of storage is being used :)